dali/kernels/test/warp_test/warp_cpu_test.cc

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/imgproc.hpp>
+#include <string>
+#include <vector>
+#include "dali/kernels/imgproc/warp_cpu.h"
+#include "dali/kernels/imgproc/warp/affine.h"
+#include "dali/kernels/test/tensor_test_utils.h"
+#include "dali/kernels/test/dump_diff.h"
+#include "dali/kernels/test/mat2tensor.h"
+#include "dali/kernels/test/test_tensors.h"
+#include "dali/kernels/scratch.h"
+#include "dali/kernels/alloc.h"
+#include "dali/test/dali_test_config.h"
+#include "dali/core/geom/transform.h"
+#include "dali/kernels/test/warp_test/warp_test_helper.h"
+
+namespace dali {
+namespace kernels {
+
+inline void IsWarpKernelValid() {
+  check_kernel<WarpCPU<AffineMapping2D, 2, float, uint8_t, float>>();
+}
```

Comment:
In my opinion, it would be more readable, if this was actually a test:
```
TEST(WarpCPU, check_kernel_compile_time) {
  check_kernel<WarpCPU<AffineMapping2D, 2, float, uint8_t, float>>();
  SUCCEED(); // I'd put it here, to make it clear, that this tests passes if compiles.
}
```

Reply:
Fair enough.